ibis/formats/pandas.py

```diff
@@ -363,6 +363,11 @@ def convert(values):
             if values is None:
                 return values
 
+            try:
+                values = iter(values)
+            except TypeError:
+                # iteration over 0-d array, happens when the element is NULL
+                return None
```

Comment:
Did you add a test case for this code path?

Reply:
Yes, test_struct_factory fails for pandas and dask without this change. That is why there was that stray forgotten commented out `xfail(["pandas", "dask"])` on that test.

Reply:
this is removed now